scio-core/src/main/scala/com/spotify/scio/io/BinaryIO.scala

```diff
@@ -47,6 +47,7 @@ final case class BinaryIO(path: String) extends ScioIO[Array[Byte]] {
         .via(new BytesSink(params.header, params.footer, params.framePrefix, params.frameSuffix))
         .withCompression(params.compression)
         .withNumShards(params.numShards)
+        .withPrefix(BinaryIO.WriteParam.DefaultPrefix)
```

Comment:
@psobot we could add this to `WriteParam` as well; this way we allow overriding if needed.

Reply:
yeah, let's keep write param parity. :+1: